src/common.rs

```diff
@@ -40,7 +40,8 @@ fn get_bin_path(version: &str) -> Option<OsString> {
 fn get_installation_path() -> Result<Option<PathBuf>> {
     if let Some(mut setting_path) = NVMD_PATH.clone() {
         setting_path.push("setting.json");
-        let setting_content = read_to_string(&setting_path)?;
+
+        let setting_content = read_to_string(&setting_path).or::<Error>(Ok("".to_string()))?;
```

Comment:
`let project_version = read_to_string(&nvmdrc).unwrap_or("".to_string());`
It would be better to use `unwrap_or` directly.

Reply:
updated